specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/stable/2022-10-01/Tables.json

```diff
@@ -705,8 +705,8 @@
           "type": "integer",
           "format": "int32",
           "minimum": 4,
-          "maximum": 2556,
-          "description": "The table total retention in days, between 4 and 2556. Setting this property to -1 will default to table retention."
+          "maximum": 4383,
+          "description": "The table total retention in days, between 4 and 4383. Setting this property to -1 will default to table retention."
         },
```

Comment:
-1 is not between 4, so how can the minimum be 4? And what does 'default to table retention' mean?

Reply:
-1 is a special value which indicate - Delete current value.